README.md

```diff
@@ -24,8 +24,8 @@
 
 [Return To Top](#nova-publish)
 
-- PHP 8.0, 8.1, 8.2
-- Laravel 8, 9, 10
+- PHP 8.2, 8.3, 8.4
+- Laravel 9, 10
```

Comment:
11 as well, right?
Maybe simplify by saying "9+"

Reply:
Ik haal Laravel weg, want die versie dwingen we niet af met Composer. Alleen Nova dwingt een bepaalde versie af, maar dat hoeft deze plugin niet te vermelden.